Resources/Prototypes/Loadouts/loadout_groups.yml

```diff
@@ -1076,12 +1076,16 @@
   loadouts:
   - HeadofSecurityJumpsuit
   - HeadofSecurityJumpskirt
-  - HeadofSecurityTurtleneck
-  - HeadofSecurityTurtleneckSkirt
+  - HeadofSecurityJumpsuitGrey # DeltaV
+  - HeadofSecurityJumpskirtGrey # DeltaV
+  - HeadofSecurityJumpsuitBlue # DeltaV
+  - HeadofSecurityJumpskirtBlue # DeltaV
+#  - HeadofSecurityTurtleneck
+#  - HeadofSecurityTurtleneckSkirt
```

Comment:
Feedback evidentially not noted, as it was for some things but not for this.

Reply:
I'm assuming you're referring to my failure to comment this out correctly...? I'll fix it next commit.